src/index.js

```diff
@@ -36,6 +43,18 @@ function Block (data) {
     }
   })
 
+  /**
+  * Creates a unique multihash key of this block.
+  *
+  * @param {string} [hashFunc='sha2-256'] - The hash function to use.
+  * @param {function} callback - The callback to execute on completion.
```

Comment:
You can indicate the function signature like this `@param {function(Error, Multihash)} callback`